collect_app/src/main/java/org/odk/collect/android/google/SheetsHelper.java

```diff
@@ -55,6 +55,13 @@
     }
 
     public void resizeSpreadSheet(String spreadsheetId, int sheetId, int columnSize) throws IOException {
+        if (sheetId < 0) {
+            throw new IllegalArgumentException("Sheet Id should be greater than or equal to 0");
```

Comment:
Any time we throw Exceptions we should detail what would cause them in a Javadoc comment. Under what conditions is one expected, etc.